src/mixins/filing-template-mixin.ts

```diff
@@ -497,9 +500,15 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
     if (!filing.correction.type) filing.correction.type = CorrectionErrorTypes.STAFF
 
     // store Correction Information
-    if (this.isCorrectedRegistration || this.isCorrectedChangeReg) {
+    if (
+      this.isCorrectedRegistration ||
+      this.isCorrectedChangeReg ||
+      this.isCorrectionFiling
+    ) {
       // Ensures startDate isn't undefined, otherwise its getter is not reactive
-      filing.correction.startDate = null
+      if (!filing.correction.startDate) {
```

Comment:
When loading the correction from a save filing.correction.startDate would already be set and this line was setting it to null, erasing the saved change.